.github/workflows/test.yml

```diff
@@ -35,9 +35,9 @@ jobs:
           targets: test
       -
         name: Upload coverage
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v5
         with:
-          file: ./coverage.txt
+          files: ./coverage.txt
```

Comment:
`file` was deprecated in favour of `files` (which now accepts a comma-separated list of files)